misc/word_palindrome.c

```diff
@@ -0,0 +1,79 @@
+/**
+ *
+ * @file
+ * @brief Check if a given word or phrase is a palindrome.
+ * @details
+ * A palindrome is a word, number, phrase, or other sequence of 
+ * characters which reads the same backward as forward, such as
+ * madam, racecar.
+ * This program will be able to check phrases, with words 
+ * separated by spaces, but can't deal with special characters.
+ * @author [Maria Eduarda Kawakami](https://github.com/madukm)
+ * @date 27 October, 2020
```

Comment:
I do not see any Doxygen sentence called `@date`.

Reply:
I fixed and pushed in my fork. Thank you for saying.